docs/en_us/platform_api/source/authentication.rst

```diff
@@ -1,23 +1,23 @@
 .. _edX API Authentication:
 
 #############################
-edX API Authentication
+EdX API Authentication
 #############################
 
-
 *************
 OAuth 2.0
 *************
 
-The edX Platform API uses OAuth 2.0 for authentication.  OAuth 2.0 is an open
+The edX Platform APIs use OAuth 2.0 for authentication. OAuth 2.0 is an open
```

Comment:
I believe that most of our APIs now support both OAuth 2.0 and session authentication. This is certainly true for the User API.